shared/chat/conversation/list.desktop.js

```diff
@@ -27,6 +27,8 @@ type State = {
   selectedMessageID?: MessageID,
 }
 
+const scrollbarWidth = 20
```

Comment:
Should we in the future detect the scrollbar width? Will this be accurate for macOS users who have enabled overlay scrollbars?